### DISCUSSION TITLE
Some questions about running python3 api/index.py

### DISCUSSION THREAD

**Discussion by muzilyd** [jyjblrd/Mocap-Drones#27] [Q&A]
I have successfully configured PS3eye and installed all required dependencies as required, as shown in the following figure
<img width="391" alt="0a0517de38159dda84ca4cb0861369ec" src="https://github.com/jyjblrd/Mocap-Drones/assets/73511912/1abbe7af-315a-4368-a31c-3185162b7004">
However, when I tried to run index.py after running yarn run dev, I encountered the following error. What should I do to resolve it.
<img width="665" alt="{F935F648-5937-499c-B59C-E4114D4913C9}" src="https://github.com/jyjblrd/Mocap-Drones/assets/73511912/bbf25d3c-e00b-4295-b1d2-620396e036ab">

**Comment by cultist123** [comment]
I encountered the same issue. How did you finally resolve it?
